float32 gimbal_x
float32 gimbal_y
float32 gimbal_z
float32 vx_set
float32 vy_set
float32 wz_set
